mlflow/R/mlflow/tests/testthat/test-tracking-runs.R

```diff
@@ -145,11 +149,33 @@ test_that("mlflow_log_batch() works", {
 
   expect_setequal(
     metrics$key,
-    c("mse", "rmse")
+    c("mse", "rmse", "na_metric")
   )
   expect_setequal(
     metrics$value,
-    c(21, 42)
+    c(23, 42, 0)
```

Comment:
Hmm I am not sure if logging 0 for missing metric value is what we want. Maybe we should just raise an exception if there are missing metric values in the dataframe?

Reply:
Per offline discussion, I've updated input validation logic to throw an exception if *any* missing (`NA`) entry is detected.